### PR TITLE
MTDome: azimuth telemetry is not connected

### PR DESCRIPTION
**Pull request by areyesd14**
In this PR, azimuth telemetries were connected correctly.

### CODE REVIEW

File: love/src/components/MainTel/MTDome/MTDomeShutter.jsx [hunk 1]
```diff
@@ -55,13 +55,16 @@ export default class MTDomeShutter extends Component {
   render() {
     const width = this.props.width;
     const height = this.props.height;
+    const positionActualDomeAz = this.props.positionActualDomeAz;
+    const positionCommandedDomeAz = this.props.positionCommandedDomeAz;
```
Comment (sebastian-aranda) [inline love/src/components/MainTel/MTDome/MTDomeShutter.jsx:59]: I suggest to use object destructuring for these variables:
`const { width, height, positionActualDomeAz, positionCommandedDomeAz } = this.props;`

File: love/src/components/MainTel/MTDome/MTDomeShutter.jsx [hunk 2]
```diff
@@ -104,6 +107,7 @@ export default class MTDomeShutter extends Component {
             height="64.86"
             style={{
               transformOrigin: `50% 50%`,
+              transition: 'transform 1.5s linear 0s',
               transform: `rotateX(${this.props.positionActualLightWindScreen}deg`,
```
Comment (sebastian-aranda) [inline love/src/components/MainTel/MTDome/MTDomeShutter.jsx:111]: Is it ok to use `rotateX` here?

File: love/src/components/MainTel/MTDome/MTDomeShutter.jsx [hunk 3]
```diff
@@ -181,18 +185,22 @@ export default class MTDomeShutter extends Component {
...
-            transformBox: 'fill-box',
-          }}
-        >
-          <path
-            style={{ stroke: '#788e9b', strokeDasharray: '4 6', fill: 'none', strokeMiterlimit: 10 }}
-            d="M72.99,6.09H28.26v4.05h40.12L22.67,50.39,1.3,102.86l1.62,53.83,58.95,77.23h-23.89v4.86h27.6l1.42,1.86h96l1.42-1.86h27.6v-4.86h-23.89l58.95-77.23,1.62-53.83-21.37-52.47L161.65,10.14h40.12V6.09h-44.72l-.51-.45H73.5l-.51,.45Zm83.8,3.65V156.87H73.71V9.74h83.08Z"
-          />
-        </g>
+        {/* Dome, commanded position */}
+        {positionCommandedDomeAz != positionActualDomeAz && (
```
Comment (sebastian-aranda) [inline love/src/components/MainTel/MTDome/MTDomeShutter.jsx:189]: I suggest to use strict comparison `!==`

**Review by sebastian-aranda — CHANGES_REQUESTED** [review]

File: love/src/components/MainTel/MTDome/MTDomeShutter.jsx (hunk 1 above)
Comment (areyesd14) [reply love/src/components/MainTel/MTDome/MTDomeShutter.jsx]: Thanks!

File: love/src/components/MainTel/MTDome/MTDomeShutter.jsx (hunk 2 above)
Comment (areyesd14) [reply love/src/components/MainTel/MTDome/MTDomeShutter.jsx]: You're right! Thank you!

File: love/src/components/MainTel/MTDome/MTDomeShutter.jsx (hunk 3 above)
Comment (areyesd14) [reply love/src/components/MainTel/MTDome/MTDomeShutter.jsx]: You're right! Thank you!

**Review by sebastian-aranda — APPROVED** [review]
Looks good, thanks @areyesd14 !

**Pull request merged**